articles/vmware/vmw-how-license-red-hat.md

```diff
@@ -23,36 +23,46 @@ UKCloud offers UKCloud for VMware with a variety of operating systems to suit yo
 
 ## Options
 
-In order to make the provision of Red Hat based virtual machines as simple and quickly as possible, UKCloud offer the following options to all customers:
+To make the provision of Red Hat based virtual machines (VMs) as simple and quick as possible, UKCloud offer the following options to all customers:
 
-- UKCloud Red Hat licencing and usage billing - One of the options is for UKCloud to provide Red Hat licencing per hour based on your usage per virtual machine. This will be billed per month in addition to the cost of the virtual machine
+- UKCloud Red Hat licencing and usage billing - One of the options is for UKCloud to provide Red Hat licensing per hour based on your usage per VM. This will be billed per month in addition to the cost of the VM.
```

Comment:
Suggest changing the last sentence to: This will be retrospectively billed based on your usage per month in addition to the cost of the VM.

Reply:
Made change as suggested